AppCenterDistribute/AppCenterDistributeTests/MSDistributeCheckForUpdateTests.m

```diff
@@ -4,111 +4,278 @@
 #import "MSBasicMachOParser.h"
 #import "MSChannelGroupProtocol.h"
 #import "MSDistributePrivate.h"
+#import "MSMockKeychainUtil.h"
+#import "MSMockUserDefaults.h"
 #import "MSTestFrameworks.h"
 #import "MSUtility+StringFormatting.h"
 
 static NSString *const kMSTestAppSecret = @"IAMSECRET";
 
 @interface MSDistributeCheckForUpdateTests : XCTestCase
 
+@property(nonatomic) MSDistribute *sut;
+@property(nonatomic) id settingsMock;
+@property(nonatomic) id bundleMock;
+@property(nonatomic) id keychainUtilMock;
+@property(nonatomic) id parserMock;
+
 @end
 
 @implementation MSDistributeCheckForUpdateTests
```

Comment:
I don't see any tests that call `startUpdateOnStart:YES`, I would think there should be some.  Or do these tests verify its use some other way without calling it directly?

Reply:
Last two tests cover this scenario as those are testing from start.